src/containers/VerletLists.h

```diff
@@ -21,13 +22,13 @@ namespace autopas {
  * Cells are created using a cell size of at least cutoff + skin radius.
  * @note This class does NOT work with RMM cells and is not intended to!
  * @tparam Particle
- * @tparam ParticleCell
  * @todo deleting particles should also invalidate the verlet lists - should be
  * implemented somehow
  */
-template <class Particle, class ParticleCell>
-class VerletLists : public LinkedCells<Particle, ParticleCell> {
-  typedef VerletListHelpers<Particle, ParticleCell> verlet_internal;
+template <class Particle>
+class VerletLists : public ParticleContainer<Particle, autopas::FullParticleCell<Particle>> {
```

Comment:
delegation of linkedcells, instead of inheritance -> now inherits from ParticleContainer